### DISCUSSION TITLE
Image cropping within a masked container

### DISCUSSION THREAD

**Discussion by pushuo** [fabricjs/fabric.js#7004] [Q&A]
Please see GIF for specific function points
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f524d34624e4f573635325566593037506a5a2f67697068792e676966](https://user-images.githubusercontent.com/21018378/113978435-87646a80-9876-11eb-8958-d3c5eaf9be9c.gif)

**Comment by asturur** [comment]
@pushuo  this isn't an help forum.
Do you have a prototype? what did you do so far with that functionality?

**Reply by pushuo** [reply]
Thank you, I have solved this problem

**Reply by rkawkxkdwlq** [reply]
>  hi could you tell me how solved this problem?
